fabric-transitive-access-wideners-v1/build.gradle

```diff
@@ -0,0 +1,60 @@
+archivesBaseName = "fabric-transitive-access-wideners-v1"
+version = getSubprojectVersion(project)
+
+loom {
+	accessWidenerPath = file('src/main/resources/fabric-transitive-access-wideners-v1.accesswidener')
+}
+
+import org.objectweb.asm.ClassReader
+import org.objectweb.asm.Opcodes
+import org.objectweb.asm.Type
+import org.objectweb.asm.tree.ClassNode
+
+import java.nio.file.FileSystem
+import java.nio.file.FileSystems
+import java.nio.file.Files
+import java.nio.file.Path
+import java.util.stream.Collectors
+
+task generateAccessWidener {
+	doLast {
+		List<String> lines = file("template.accesswidener").text.lines().collect(Collectors.toCollection { [] })
+		Path inputJar = loom.namedMinecraftProvider.parentMinecraftProvider.mergedJar
+
+		try (def fs = FileSystems.newFileSystem(URI.create("jar:${inputJar.toUri()}"), [create: false])) {
+			generateBlockConstructors(lines, fs)
+		}
+
+		file('src/main/resources/fabric-transitive-access-wideners-v1.accesswidener').text = String.join('\n', lines) + '\n'
```

Comment:
It's also done manually in the data gen module so I think it's fine, but it's easy to change in any case

Reply:
I agree, its best to be done manually, it would require 2 builds, first one to generate and then a 2nd to use. Without something more specific in loom it will wont be much better than manually. Could possibly have one task to run the regen for all modules, but again not a major issue now.

Reply:
Hmm, I don't think we actually have to declare this one for loom in the first place since this module doesn't contain any code